static-analysis/assert-throws.php.disabled

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Webmozart\Assert\StaticAnalysis\AssertString;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @psalm-return callable() : never-returns
```

Comment:
Doesn't seem to work: the idea is that we want to be sure that this is a zero-argument never-returns closure (equivalent to a thrown exception)